src/pages/Swap/index.tsx

```diff
@@ -255,6 +256,18 @@ export default function Swap() {
     [onCurrencySelection]
   )
 
+  const FEE_MESSAGE_CURRENCIES_CODE = ["ETH"]
+
+  const inputCurrency = currencies[Field.INPUT]?.symbol || ""
+  const outputCurrency = currencies[Field.OUTPUT]?.symbol || ""
+  
+  let currencyMessage = ""
+  if (FEE_MESSAGE_CURRENCIES_CODE.includes(inputCurrency)) { 
+    currencyMessage = inputCurrency
+  } else if (FEE_MESSAGE_CURRENCIES_CODE.includes(outputCurrency)) {
+    currencyMessage = outputCurrency
+  }
```

Comment:
I think a more robust approach here would be to check first if the selected network is polygon by using the network id (which should be 137 for polygon). You can use this `chainId` variable for that https://github.com/1Hive/honeyswap-interface/blob/master/src/pages/Swap/index.tsx#L79 and then, if the selected network is polygon, compare both token contracts with WETH contract, instead of the symbols (WETH contract should be this https://info.honeyswap.org/#/token/0x7ceb23fd6bc0add59e62ac25578270cff1b9f619)

Reply:
We (me and @hernandoagf) ended up agreeing on just making the conditional with chainId, it seems unlikely there's an error using the token symbol to make the conditional to display the message.